lib/ProMotion/table/cell/table_view_cell_module.rb

```diff
@@ -89,12 +100,26 @@ def set_accessory_type
       self.accessoryType = map_accessory_type_symbol(data_cell[:accessory_type]) if data_cell[:accessory_type]
     end
 
+    def prepareForReuse
+      if @remote_image_operation && @remote_image_operation.respond_to?(:cancel)
+        @remote_image_operation.cancel
+        @remote_image_operation = nil
+      end
+      self.send(:prepare_for_reuse) if self.respond_to?(:prepare_for_reuse)
```

Comment:
good call. I'm jsut finishing the tests to make sure that this method calls `prepare_for_reuse` on the cell and that `prepare_for_reuse` is called BEFORE `on_reuse`